utils/utils.jsx

```diff
@@ -872,86 +820,27 @@ export function applyTheme(theme) {
             'sidebar-unread-text-rgb': toRgbValues(theme.sidebarUnreadText),
 
             // Hex CSS variables
-            // TODO: phase out changeOpacity() here
             'sidebar-bg': theme.sidebarBg,
             'sidebar-text': theme.sidebarText,
-            'sidebar-text-08': changeOpacity(theme.sidebarText, 0.08),
```

Comment:
Have you checked with the teams developing plugins to see if they're using any of these? If compatibility is a problem, perhaps we can leave these variables around until 6.0, include them in the deprecation list, and loudly yell at anyone who tries to use them in the web app in the mean time?

Reply:
Excellent point, let's put them back! 🙂